.github/workflows/release.yml

```diff
@@ -266,3 +266,22 @@ jobs:
     - name: Wait for Metabase to start
       run: while ! curl -s 'http://localhost:3000/api/health' | grep '{"status":"ok"}'; do sleep 1; done
       timeout-minutes: 3
+
+  publish-elastic-beanstalk-artifacts:
+    runs-on: ubuntu-20.04
```

Comment:
```suggestion
    runs-on: ubuntu-22.04
```

Reply:
Thanks for spotting this! I've changed it accordingly.